sceptre/scipool/config/prod/scheduledjobsvpc.yaml

```diff
@@ -11,3 +11,4 @@ dependencies:
 parameters:
   VpcSubnetPrefix: "10.255.40"
   VpcName: "scheduledjobsvpc"
+  IncludeBastianSecurityGroup: "false"
```

Comment:
Bastian SG should be removed for all `scheduledjobsvpc` so could you please make this change to all files under `sceptre/scipool/config` folder ?